addons/account/static/src/components/tax_totals/tax_totals.xml

```diff
@@ -24,16 +24,20 @@
                     <t t-else="">
                         <span class="o_tax_group_edit" t-on-click.prevent="() => this.setState('edit')">
                             <i class="fa fa-pencil me-2"/>
-                            <span class="o_tax_group_amount_value">
-                                <t t-out="props.taxGroup.formatted_tax_group_amount"/>
-                            </span>
+                            <div class="o_list_monetary">
```

Comment:
You should be able to remove the need for additional divs if you set the span's display to block (their default display value is inline, which is why text-align doesn't work). I then suggest you modify the css like below:
```css
span.o_list_monetary {
  display: block
}
.o_list_monetary {
  ...
}
```
I think this would make the html cleaner, let me know what you think about it and if it works properly.

Reply:
Thanks for the suggestion , it works properly